src/category_theory/category/PartialFun.lean

```diff
@@ -135,6 +135,7 @@ equivalence.mk PartialFun_to_Pointed Pointed_to_PartialFun
       unfold_projs,
       dsimp,
       change option.elim _ _ _ = _,
+      dsimp,
```

Comment:
This pattern looks slightly strange to me - you have three lines each changing the syntactic but not definitional form of the goal. Does it work to just `change` directly to the helpful version?

Reply:
IIRC I wrote this originally to avoid writing down a huge `change`, because the goal is three lines long at this point in the proof.